rholang/test.sh

```diff
@@ -0,0 +1,10 @@
+sbt bnfc:generate
+sbt assembly
+# Run compiler over all test rholang files
+for i in tests/*; do
+	./rho2rbl "$i"
+done
+# Rewrite all tests to print to a text file results.txt instead of stdout
```

Comment:
This is an impressive kludge.

Reply:
I thought this was the simplest way to handle things. Any better ideas?

Reply:
It's the simplicity that's impressive! :D The "right" way would be to have the VM able to read commandline flags and pass them into contracts to use; then you'd have the compiler output `ostream-print (ostream-new args[0])` directly.  But this is fine for the moment.